### PR TITLE
using firebase on sharedPreferences

### PR DESCRIPTION
**Pull request by ElioNeto**
using firebase database (noSql json based) for saves list of preferences of the users (my english is so bad haha).
You need make a project on firebase and get apikeys. 
If you need, I can help you on the proccess.
Send a email from netoo.elio@hotmail.com or call me on the facebook (fb.com/elio.neto1)

### CODE REVIEW

**Comment by carolinaknoll** [comment]
Hey there, thank you for contributing with Firebase! 😄 

However, there will be some changes in the way that data is saved, so I would prefer to add this integration in a later time. In any case, I will leave this open for now, so that I can use your help when needed.

Thank you! 😄

**Review by carolinaknoll — APPROVED** [review]

File: src/components/topic-list.js [hunk 1]
```diff
@@ -141,9 +207,19 @@ export default class TopicList extends Component {
 	render() {
     return (
       <div className="topic-list-container">
-      {this.props.clickedTopic ? (
-        this.renderTopicList()
-      ) : null}
+        {/* <p>{this.props.clickedTopic}</p> */}
```
Comment (katherine95) [inline src/components/topic-list.js:210]: @ElioNeto You can get rid of commented code.

**Comment by katherine95** [comment]
Great job @ElioNeto. Otherwise, if you didn't add any new package revert changes in the `yarn.lock` file.

**Comment by carolinaknoll** [comment]
Hello again. 😄 

As I don't currently have plans on adding Firebase to the project, I will be closing this PR for now. However, thank you for spending your time to contribute to this project. It was nice to be able to see how it could be implemented, and also handle auth, login and user info.

Cheers ☕️

**Pull request closed without merging**